deploy/eck-operator/values.yaml

```diff
@@ -188,6 +193,8 @@ config:
       #       serviceMonitorNamespaceSelector: {}
       #       serviceMonitorSelectorNilUsesHelmValues: false
       enabled: false
+      # additional volume mounts for the kube-rbac-proxy container.
+      volumeMounts: []
```

Comment:
IIUC, if you want to disable automount service account tokens and secure the metrics entrypoint, you will need to define `volumeMounts` twice?

```yaml
volumeMounts:
- mountPath: /var/run/secrets/kubernetes.io/serviceaccount
  name: serviceaccount-token
  readOnly: true

config:
  metrics:
    secureMode:
      volumeMounts:
      - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
        name: serviceaccount-token
        readOnly: true
  ```

Reply:
I'm missing something. In which case, would we have two different `volumeMounts`?

Reply:
If `automountServiceAccountToken: false` and both the `kube-rbac-proxy` and `manager` containers require to talk to the K8s api we would need to explicitly set in each one of the containers the extra `volumeMounts` with the `serviceaccounttoken` mounted in order to reach K8s api.

Unless I misunderstood and `kube-rbac-proxy` container actually does not require to communicate with the K8s API, in which case the extra `volumeMount` is not required for it 😅 

TLDR: Extra `volumeMounts` should be added to any container still requiring K8s API access in case `automountServiceAccountToken` is `false`

Reply:
I was a bit stuck in my mind with a single definition of volumesMounts for all containers, but there may be cases where you want different additional volumesMounts for a given container, which makes the need to have separate definitions valid per container.